partials/archive-exhibitor/apply.php

```diff
@@ -13,7 +13,7 @@
       if (!empty($exhibitors_apply_heading)) {
     ?>
       <div class="row">
-        <div class="col col-s col-s-12 text-align-center">
+        <div class="col col-s col-s-12 col-m col-m-12 col-l col-l-12 text-align-center">
```

Comment:
so the 12 col classes dont do anything. they override a value of 100% with a new value of 100%. but the `col-m` and `col-l` classes change the gutter width. why not just override the gutter values for `.col` to save every column needing 4 `col` declarations?